src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Presentation/SampleChooserViewModel.cs

```diff
@@ -740,6 +761,8 @@ private async Task<List<SampleChooserContent>> GetFavoriteSamples(CancellationTo
 		/// <returns>The updated content</returns>
 		public async Task<object> UpdateContent(CancellationToken ct, SampleChooserContent newContent)
 		{
+			Console.WriteLine("UpdateContent Creating Instance");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6226524)